Plugins/Alpakit/Source/Alpakit/Private/AlpakitModEntryList.cpp

```diff
@@ -108,3 +120,48 @@ void SAlpakitModEntryList::SetShowEngine(bool bInShowEngine) {
 	bShowEngine = bInShowEngine;
 	LoadMods();
 }
+
+FReply SAlpakitModEntryList::PackageAllMods() {
+	TSharedPtr<SAlpakitModEntry> first;
+	TArray<TSharedPtr<SAlpakitModEntry>> nextEntries;
+
+	UE_LOG(LogTemp, Display, TEXT("Alpakit All!"))
+
+	for (auto mod : FilteredMods) {
```

Comment:
do not use auto, use proper type. Also, here you are implicitly copying mod object, because you're using auto instead of proper reference type. 
Also, codestyle. Variable names start with uppercase.